src/Components.sol

```diff
@@ -1,13 +1,13 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.2 <0.9.0;
 
-import "./console.sol";
-import "./console2.sol";
-import "./StdAssertions.sol";
-import "./StdCheats.sol";
-import "./StdError.sol";
-import "./StdJson.sol";
-import "./StdMath.sol";
-import "./StdStorage.sol";
-import "./StdUtils.sol";
-import "./Vm.sol";
+import "src/console.sol";
+import "src/console2.sol";
+import "src/StdAssertions.sol";
+import "src/StdCheats.sol";
+import "src/StdError.sol";
+import "src/StdJson.sol";
+import "src/StdMath.sol";
+import "src/StdStorage.sol";
+import "src/StdUtils.sol";
+import "src/Vm.sol";
```

Comment:
I'm hesitant to make this change, just because we had some issues with absolute paths in the past and --base-url is not supported by older solc versions

Reply:
Was https://github.com/foundry-rs/foundry/issues/3440 ever resolved, and are there other remaining issues related to this?

> and --base-url is not supported by older solc versions

It looks like this was [introduced](https://github.com/ethereum/solidity/blob/develop/Changelog.md#069-2020-06-04.) in 0.6.9. So does that imply paths of this format won't work with projects using solc versions less than that? If so happy to switch back to relative paths for forge-std here (even though these fully qualified paths are definitely superior IMO 😛)

Reply:
Updated to use relative paths